openblas-libs/build.sh

```diff
@@ -0,0 +1,5 @@
+#/bin/bash
+
+make USE_THREAD=0 DYNAMIC_ARCH=1 NO_STATIC=1 PREFIX=$PREFIX
```

Comment:
Instead of setting `USE_THREAD=0`, could you set `NO_AFFINITY=1 NUM_THREADS=1`. This will default to 1 thread for all practical purposes, but allow people to override if they wish by setting the environment variable `OPENBLAS_NUM_THREADS` or one of the other ones if they are relevant ( <https://github.com/xianyi/OpenBLAS#set-the-number-of-threads-with-environment-variables> ) before loading the library. The `NO_AFFINITY` is necessary to avoid issues when forking. I have also found this works well on certain flavors of Linux that otherwise run into trouble. Also, it avoids an issue with building R ( https://stat.ethz.ch/pipermail/r-sig-hpc/2012-April/001348.html ).